src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -58,9 +72,12 @@ public static CLRModuleType GetInstance(Type type, CLRDLLModule module, string a
                     {
                         mtype = new CLRModuleType(type);
                         //Now check that a type with same name is not imported.
+                        bool equivalentComTypes = false;
                         Type otherType;
                         if (mTypeNames.TryGetValue(mtype.FullName, out otherType))
+                        {
                             throw new InvalidOperationException(string.Format("Can't import {0}, {1} is already imported as {2}, namespace support needed.", type.FullName, type.Name, otherType.FullName));
```

Comment:
If no other ideas work, I suggest that we only change the message:
if (type.IsEquivalentTo(otherType)) throw "Equivalent Embeded interop types detected..etc. Dynamo does not support ...etc "
ANd maybe remove the `namespace support needed` part of the message.

FYI @mjkkirschner